action.yml

```diff
@@ -13,6 +13,9 @@ inputs:
       - run
       - pre-job
       - post-job
+  features:
+    description: A comma separated list of features to enable. Example scanning, mapping, all. Defaults to scanning.
+    default: scanning

```

Comment:
Wasn’t this supposed to be ‘all’?

Reply:
Chatted offline, defaulting to scanning until post public preview.